lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -86,3 +86,12 @@
     display: flex;
     justify-content: space-between;
 }
+
+.search-help-tutorial-1,
```

Comment:
These were the minimum styles necessary for the help tutorial. Additional styling of the help tutorial will be included in the styling search page PR.